OpenSim/Common/Component.cpp

```diff
@@ -1531,7 +1531,8 @@ void Component::printInputInfo() const {
                   << std::string(maxlenInputName -
                                  input->getName().length(), ' ')
                   << input->getName() << " : ";
-        if (input->getNumConnectees() == 0) {
+        if (input->getNumConnectees() == 0 || 
+            (input->getNumConnectees() == 1 && !input.isConnected())) {
```

Comment:
If parsing an XML file, I think that assumption is correct, because the parser should trim whitespace. That is, 

```
<input_foo_connectee_name>a b       c</input_foo_connectee_name>
```

should be interpreted as the list `["a", "b", "c"]`.

However, if setting the connectee names programmatically through the API, the assumption might not hold. I think that would be a separate bug, though (`setConnecteeName()` should validate the provided name).